aliases/available/clipboard.aliases.bash

```diff
@@ -1,20 +1,19 @@
+# shellcheck shell=bash
 cite 'about-alias'
-about-alias 'pbcopy and pbpaste shortcuts to linux'
+about-alias 'xclip shortcuts'
 
-case $OSTYPE in
-  linux*)
-    XCLIP=$(command -v xclip)
-    [[ $XCLIP ]] && \
-    alias pbcopy="$XCLIP -selection clipboard" && \
-    alias pbpaste="$XCLIP -selection clipboard -o"
-    ;;
-esac
+if _command_exists xclip; then
```

Comment:
Actually I think that we do not need to check it... we should not check for it when setting up aliases, just like we do not do it in the `git` alias file

Reply:
fixed